bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -83,6 +83,10 @@ type ksServer struct {
 	// other information about the regisry.
 	knownRegistries map[string]RegistryConfig
 
+	//GKE version to use, suggest format 1.X.Y, avoid using most recent version
```

Comment:
Please change the comment and variable name. I'd like to convey the fact that this should only be used as a temporary override when things are going wrong. 

so lets call it something like
gkeVersionOverride

and the comment should indicate that  this should only be used temporarily if we need to override the default value in the DM config file.

The DM config file should be the source of truth. We want to avoid setting defaults in multiple places. That will lead to all kinds of confusion. 

For example, we will upgrade the DM configs to use GKE 1.11 and make other changes based on 1.11; this will work with CLI and then break with the web app because the web app will be overriding the GKE version.

If we set this flag the procedure should be to file an issue to unset it and revert to the default value being set in the DM config.

Reply:
Can you fix the comment. 
Delete "GKE version to use, suggest format 1.X.Y, avoid using most recent version"

Change it to something like

"gkeVersionOverride allows overriding the GKE version specified in the DM config. If not set the value in the DM config is sued."